Sources/XMTPiOS/Conversation.swift

```diff
@@ -359,4 +359,13 @@ public enum Conversation: Identifiable, Equatable, Hashable {
 			return try dm.isActive()
 		}
 	}
+    
```

Comment:
Think this could use a comment just mentioning that the Dictionary returned uses inbox_ids of conversation participants as keys.